gst/nnstreamer/tensor_typedef.h

```diff
@@ -30,6 +30,7 @@
 
 #include <stddef.h>
 #include <stdint.h>
+#include <glib.h>
```

Comment:
Do not include glib in commin external headers so that external developers can write custom filters without depending on glib. Use char instead of gchar in this header.

Reply:
it's applied